pkg/policy/api/cidr.go

```diff
@@ -35,7 +35,7 @@ type CIDRRule struct {
 
 	// CIDRGroupRef is a reference to a CiliumCIDRGroup object.
 	// A CiliumCIDRGroup contains a list of CIDRs that the endpoint, subject to
-	// the rule, can (Ingress) or cannot (IngressDeny) receive connections from.
+	// the rule, can (Ingress/Egress) or cannot (IngressDeny/EgressDeny) receive connections from.
```

Comment:
> To account for Egress and EgressDeny

It turned out that we were actually missing support for CIDRGroupRef in IngressDeny, despite the docstring mentioning it (thanks to @gandro for the heads up). I opened a PR to fix that: https://github.com/cilium/cilium/pull/30933

Regarding this PR, I think this is fine as is, we just need to remove the "EgressDeny" part added here. Then, we can add support for EgressDeny in a follow up PR (IMHO this is better to avoid ending up with a PR too large).

Could you please remove the reference to `EgressDeny` here? Sorry for the confusion :pray:

Reply:
Updated.